PACKAGING.md

```diff
@@ -23,10 +23,11 @@ The Agama packages are available in two OBS projects:
 You can find more details the automatic OBS synchronization in the
 [obs_integration.md](doc/obs_integration.md) file.
 
-The process to build each package is slightly different depending on the technology we are using.
-While the Ruby-based one (`rubygem-agama`) is built as any other YaST package, the web UI
-(`cockpit-agama`) and the CLI (`agama-cli`) rely on [OBS source
-services](https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.source_service.html).
+The process to build each package is slightly different depending on the
+technology we are using. While the Ruby-based one (`rubygem-agama-yast`) is
+built as any other YaST package, Agama server (`agama`), the CLI (`agama-cli`),
```

Comment:
well, it is kind of misleading as we do not build other YaST packages as rubygems...so I would say as we package it similar to other YaST packages and builds it as common rubygem.